mayastor/src/replica.rs

```diff
@@ -338,18 +316,13 @@ pub(crate) async fn share_replica(
     // shared
     if replica.get_share_type().is_none() {
         match want_share {
-            rpc::ShareProtocolReplica::ReplicaIscsi => replica
-                .share(ShareType::Iscsi)
-                .await
-                .context(ShareReplica {
-                    uuid: args.uuid.clone(),
-                })?,
             rpc::ShareProtocolReplica::ReplicaNvmf => {
                 replica.share(ShareType::Nvmf).await.context(ShareReplica {
                     uuid: args.uuid.clone(),
                 })?
             }
-            rpc::ShareProtocolReplica::ReplicaNone => (),
+            rpc::ShareProtocolReplica::ReplicaIscsi
```

Comment:
Do we need to keep this whilst removing iSCSI support ?

Reply:
Yes, that's auto-generated from the RPC protofile and is simply needed here to satisfy the compiler that all enum values have been handled.